packages/back-end/src/app.ts

```diff
@@ -347,6 +347,10 @@ app.get(
 );
 app.get("/experiments/newfeatures/", experimentsController.getNewFeatures);
 app.get("/experiments/snapshots/", experimentsController.getSnapshots);
+app.post(
```

Comment:
Any reason we're using `post` instead of `get`? It seems like the endpoint is doing a lookup and not creating anything

Reply:
We need to pass a trackingKey into this endpoint that may contain non-url safe characters.  I did it via a POST request body, but we could switch it to use a GET request and a querystring instead.